pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -141,10 +150,7 @@ public IngestionDelayTracker(ServerMetrics serverMetrics, String tableNameWithTy
    *
    * @param ingestionTimeMs original ingestion time in milliseconds.
    */
-  private long getIngestionDelayMs(Long ingestionTimeMs) {
-    if (ingestionTimeMs == null) {
-      return 0; // return 0 when not initialized
-    }
+  private long getIngestionDelayMs(long ingestionTimeMs) {
```

Comment:
Suggest adding a checking to prevent invalid value:
```
  if (ingestionTimeMs <= 0) {
    return 0;
  }
```

Reply:
Same as above set to < 0 so as to take 0 as valid timestamp, see commit 9bb0befe9a55c01bb3a5066a4b30137469819f38